Makefile

```diff
@@ -9,7 +9,7 @@
 2BPP    := rgbgfx
 
 ASM     := rgbasm
-ASFLAGS := --export-all
```

Comment:
As this is part of a script, I slightly prefer to use long option names (such as `--nop-after-halt`) : it is easier to understand for casual readers.